manifests/prometheus/alerts.d/aiven-cost.yml

```diff
@@ -6,18 +6,11 @@
   value:
     name: AivenEstimatedCostHigh
     rules:
-      - &alert
-        alert: AivenEstimatedCostHigh_Warning
-        expr: paas_aiven_estimated_cost_pounds > 2600
+      - alert: AivenEstimatedCostHigh
+        expr: paas_aiven_estimated_cost_pounds > 6500
```

Comment:
I've been thinking about changing the alert expr here to `delta(paas_aiven_estimated_cost_pounds[1d])*30 > 0` or something, so we’re not alerting on a sawtooth.

That data looks like this:

![image](https://user-images.githubusercontent.com/1696784/51839376-fd690080-2300-11e9-99d2-4403b15a0f6c.png)

... so I wonder if 6,500 is still a little too low?

Reply:
I think that makes sense, but I think it should also be a separate story. This was intended to be a quick fix to get the board green while we work out what the longer-term solution to this alerting is.